step.yml

```diff
@@ -1,11 +1,11 @@
 title: Run CocoaPods install
 summary: |-
-  You don't need to keep the installed pods in your repo. Simply add this Step
-  and Bitrise will do the pod install for you on the VM!
+  You don't need to keep the installed Pods in your repo. Simply add this Step
+  and Bitrise will do the Pod install for you on the VM!
 description: |-
 
-  CocoaPods is a dependency manager for Swift and Objective-C projects. This Step uses CocoaPods' `pod install` command to install your dependencies on the virtual machine where your Bitrise build runs.   
-  CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If your Gemfile.lock file contains the `cocoapods` gem then the step will call `pod install` with `bundle exec`. Otherwise, the Cocoapods version in the Podfile.lock will be installed as a global gem.
+  CocoaPods is a dependency manager for Swift and Objective-C projects. This Step uses CocoaPods' `pod install` or `pod update` command to install your dependencies on the virtual machine where your Bitrise build runs.   
+  CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If your Gemfile.lock file contains the `cocoapods` gem then the step will call the pod install command with `bundle exec`. Otherwise, the Cocoapods version in the Podfile.lock will be installed as a global gem.
```

Comment:
CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If your Gemfile.lock file contains the `cocoapods` gem, then the Step will call the `pod install` command with `bundle exec`.
(added a comma after gem, step > Step, and marked pod install as a command with `..`)